src/js/components/Banner.js

```diff
@@ -1,88 +1,62 @@
 /* globals BANNERS_URL */
 import React from 'react';
-import CrossIcon from './Icons/Times';
-import cx from 'classnames';
-import { saveToLocalStorage, getBooleanFromLocalStorage } from '@/util';
+
+import { saveToLocalStorage, getStringFromLocalStorage } from '@/util';
 import { dateMath } from '../util/index.js';
+import CrossIcon from './Icons/Times';
+
 export default class Banner extends React.PureComponent {
 
   state = {
-    title: '',
-    message: '',
-    toggleBannerVisibilty: false,
+    notifications: [],
     date: 0,
-    index: 0,
   };
 
   componentDidMount() {
-    const d = new Date();
-    const $mysqlDate = d.getFullYear() + '-' + (d.getMonth() + 1) + '-' + d.getDate() + ' ' + d.getHours() + ':' + d.getMinutes() + ":" + d.getSeconds();
-    const $date = $mysqlDate.slice(0, 10);
+    const currentDate = new Date();
+    const date = currentDate.toISOString().slice(0, 10);
```

Comment:
`const date = currentDate.toISOString().slice(0, 10);`
The toISOString function will auto go to the internation dateline time and will therefore not show messages based on user's local time. Might be a problem

Reply:
Ah! got it. Will update. Thanks :)